scripts/variantstore/beta_docs/gvs-quickstart.md

```diff
@@ -116,16 +116,16 @@ To run the GVS workflow on your own sample data, follow the instructions in the
 
 By default, the workflow is set up to write outputs to the workspace Google bucket. If you want to write the outputs to a different cloud storage location, you can specify the cloud path in the `extract_output_gcs_dir` optional input in the workflow configuration. 
 
-### Time and cost estimates
-Below is an example of the time and cost of running the workflow.
+### Time and cost
+Below is an example of the time and cost of running the workflow with the sample data pre-loaded in the workspace.
 
-| Number of Samples | Wall Clock Time | Cost $ | Cost per Sample |
-| ---  | --- | --- | --- |
-| 10 | 04:30:00 | $0.84 | ~$0.08 |
-| 1528 |
-| 10379 | 05:38:00 | $683.33 | $0.0658 |
+| Number of Samples | Elapsed Time (hh:mm) | Terra Cost | BigQuery Cost | Total Cost | Approximate Cost per Sample |
+|-------------------|----------------------|------------|---------------|------------|-----------------------------|
+| 10                | 04:30                | $0.84      | $0.51         | $1.35      | $0.14                       |
```

Comment:
I thought we had run this on other sets that were larger than this to provide more cost information?

Reply:
Ah I see those are in a different document and this is just for the quickstart. Nvm!